### ISSUE TITLE
Install on OSX Broken

### ISSUE THREAD

**Issue by friedmud** [mit-crpg/PINSPEC#75]
Not quite sure what's up here.  On OSX PINSPEC builds just fine (no obvious errors), but attempting to run any of the sample-input files just yields:

```
  File "doppler.py", line 2, in <module>
    from pinspec import *
  File "/Users/gastdr/.local/lib/python2.7/site-packages/pinspec/__init__.py", line 13, in <module>
    set_log_level('NORMAL')
NameError: name 'set_log_level' is not defined
```

I tried to go into **init**.py and comment that out... but then it just gets stuck on the next thing.  It seems to me that something about swig isn't working... like the python interface files aren't getting created.

Any ideas?

**Comment by cjosey** [comment]
I used to have a similar issue and I think I remedied it by just running
the install script twice in a row.

**Comment by wbinventor** [comment]
Yes, this is a common problem for swig-wrapped applications installed with
distutils. Since PINSPEC is sort of an odd educational tool, I haven't
spent time upgrading the build system so I apologize for the inconvenience.
BTW, what are you using PINSPEC for?

**Comment by friedmud** [comment]
@cjosey running the install twice DID fix the problem!  Awesome!  Thanks!

@wbinventor Ben has us using it a bit for generating XS for 212

If you would like to close this issue - that's fine with me...

**Comment by wbinventor** [comment]
I don't plan to fix this so you can close it unless you care to submit a PR.

**Comment by wbinventor** [comment]
But I wouldn't worry about that since from what I recall after having
looked at this before, it's not an easy problem to solve :-(
